controllers/admin/AdminImportController.php

```diff
@@ -4260,108 +4264,108 @@ protected function supplyOrdersDetailsImportOne($info, &$products, &$reset, $for
                 Tools::htmlentitiesUTF8($info['supply_order_reference']),
                 $current_line + 1
             );
+
+            return;
         }
 
-        if (empty($this->errors)) {
-            // sets parameters
-            $id_product = (int) $info['id_product'];
-            if (!$info['id_product_attribute']) {
-                $info['id_product_attribute'] = 0;
-            }
-            $id_product_attribute = (int) $info['id_product_attribute'];
-            $unit_price_te = (float) $info['unit_price_te'];
-            $quantity_expected = (int) $info['quantity_expected'];
-            $discount_rate = (float) $info['discount_rate'];
-            $tax_rate = (float) $info['tax_rate'];
-
-            // checks if one product/attribute is there only once
-            if (isset($products[$id_product][$id_product_attribute])) {
-                $this->errors[] = sprintf(
-                    $this->trans('Product/Attribute (%d/%d) cannot be added twice (at line %d).', [], 'Admin.Advparameters.Notification'),
-                    $id_product,
-                    $id_product_attribute,
-                    $current_line + 1
-                );
-            } else {
-                $products[$id_product][$id_product_attribute] = $quantity_expected;
-            }
+        // sets parameters
+        $id_product = (int) $info['id_product'];
+        if (!$info['id_product_attribute']) {
```

Comment:
why not `!isset($info['id_product_attribute'])` ?

Reply:
Fixed